src/components/CuratorDashboard/CuratorProfile/CuratorProfileCard.tsx

```
@@ -0,0 +1,314 @@
+				}
+			}
+		});
+		setOnChainIdentity(onChainIdentity);
+	};
+
+	useEffect(() => {
+		if (!api || !apiReady || !address) return;
+		let unsubscribes: (() => void)[];
+		fetchIdentityInformation();
+		return () => {
+			unsubscribes && unsubscribes.length > 0 && unsubscribes.forEach((unsub) => unsub && unsub());
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)